include/seqan3/argument_parser/validators.hpp

```diff
@@ -83,12 +83,11 @@ namespace seqan3
 //!\cond
 template <typename validator_type>
 SEQAN3_CONCEPT validator = std::copyable<remove_cvref_t<validator_type>> &&
-                           requires(validator_type validator,
-                                    typename std::remove_reference_t<validator_type>::value_type value)
+                           requires(validator_type validator)
 {
     typename std::remove_reference_t<validator_type>::value_type;
 
-    { validator(value) } -> void;
+    { validator(typename std::remove_reference_t<validator_type>::value_type{}) } -> void;
```

Comment:
I would use decval in case the value type is not default constructible?

Reply:
Yes, but it doesn't wark because GCC7 and GCC8 at least trigger ODR-use of that statement (although they shouldn't) which results in the compiler telling you that you cannot use declval in an evaluated context. :man_shrugging:

Reply:
:expressionless: really...

Reply:
yeah I've seen this as well, which is annoying.